hieunc-guestbook-dojo/static/js/guestbook/widget/WidgetGuestBookList.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Created by hieunc on 25/11/2015.
+ */
+
+define([
+	"dojo/_base/declare",
+	"dojo/_base/lang",
+	"dojo/_base/array",
+	"dojo/topic",
+	"dojo/dom-construct",
+	"dojo/on",
+	"dojo/text!./templates/WidgetGuestBookList.html",
+	"guestbook/widget/_base/_ViewBaseMixin",
+	"guestbook/store/GreetingStore",
+	"guestbook/widget/WidgetGuestBookGreeting",
+	"guestbook/widget/WidgetGuestBookSign"
+], function (declare, lang, array, topic, domConstruct, on, template, _ViewBaseMixin, GreetingStore, WidgetGuestBookGreeting,
+             WidgetGuestBookSign) {
+	return declare("guestbook.widget.WidgetGuestBookList", [_ViewBaseMixin], {
+		templateString: template,
+
+		constructor: function (/*Object*/ kwArgs) {
+			this.inherited(arguments);
+		},
+
+		postCreate: function () {
+			this.inherited(arguments);
+
+			var guestbookSignWidget = new WidgetGuestBookSign({
+				guestbook_name: this.GuestBookNameNode.value
+			});
+
+			domConstruct.place(guestbookSignWidget.domNode, this.GuestBookSignNode);
+			var fnGetList = lang.hitch(this, "getList");
+			fnGetList();
+			this.own(
+					on(this.btnGetListNode, "click", fnGetList),
+					topic.subscribe("refreshList/topic", function (e) {
```

Comment:
var fnGetList = lang.hitch(this, "getList"); do dung 2 lan nen em khai bao dung chung